shelldio.sh

```diff
@@ -201,6 +234,9 @@ for binary in grep curl info sleep clear killall; do
 	fi
 done
 
+# Έλεγχος εγκυρότητας λίστας σταθμών
+validate_station_lists
```

Comment:
```shell
$ ./shelldio.sh
awk: can't open file /Users/drpaneas/.shelldio/all_stations.txt
 source line number 1
Πρόβλημα: Η λίστα σταθμών: /Users/drpaneas/.shelldio/all_stations.txt δεν είναι έγκυρη
Εκτέλεσε shelldio --fresh για να κατεβάσεις τη λίστα εκ νέου
```

Η `validate_station_lists` θα πρέπει να καλείται **μετά** από το κομμάτι του κώδικα που βεβαιώνει ότι υπάρχει το αρχείο για το οποίο θα κάνει validate.

Reply:
Αρχικά το μετέφερα πιο κάτω.

Αλλά στη συνέχεια υλοποίησα αυτό:
> Πριν ελέγξεις την εγκυρότητα, έλεγξε την ύπαρξη.

Οπότε το επέστρεψα στην αρχή, μετά τα binaries check. Το αρχείο που θα έρχεται από το repo πρέπει να είναι valid. (προτείνω ένα github action να τσεκάρει το csv για validity)

Reply:
> Οπότε το επέστρεψα στην αρχή, μετά τα binaries check. Το αρχείο που θα έρχεται από το repo πρέπει να είναι valid. (προτείνω ένα github action να τσεκάρει το csv για validity)

Ok καλή ιδέα. Έχεις κάποιο στα υπόψιν σου να προσθέσω ή να βάλω όποιο νάνε ?

Reply:
> > Οπότε το επέστρεψα στην αρχή, μετά τα binaries check. Το αρχείο που θα έρχεται από το repo πρέπει να είναι valid. (προτείνω ένα github action να τσεκάρει το csv για validity)
> 
> Ok καλή ιδέα. Έχεις κάποιο στα υπόψιν σου να προσθέσω ή να βάλω όποιο νάνε ?

Not really. Δεν ξέρω αν υπάρχει κάποιο out-of-the-box. Μπορούμε να γράψουμε το δικό μας αν χρειαστεί.

Reply:
> > > Οπότε το επέστρεψα στην αρχή, μετά τα binaries check. Το αρχείο που θα έρχεται από το repo πρέπει να είναι valid. (προτείνω ένα github action να τσεκάρει το csv για validity)
> > 
> > 
> > Ok καλή ιδέα. Έχεις κάποιο στα υπόψιν σου να προσθέσω ή να βάλω όποιο νάνε ?
> 
> Not really. Δεν ξέρω αν υπάρχει κάποιο out-of-the-box. Μπορούμε να γράψουμε το δικό μας αν χρειαστεί.

fixed: 1b8a1c231e674861938b4bd76fa06030f6ecf761
issue: https://github.com/CerebruxCode/shelldio/issues/62